src/components/reusable/progressBar/progressBar.scss

```diff
@@ -0,0 +1,212 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+$progress-bar-status-colors: (
+    default: var(--kd-color-background-primary-hover),
+    success: var(--kd-color-background-success),
+    error: var(--kd-color-background-error)
+);
+
+$progress-bar-height: 4px;
+$progress-bar-radius: 4px;
+$progress-bar-background: var(--kd-color-background-inverse-hover);
+$progress-bar-indeterminate-color: map-get($progress-bar-status-colors, default);
+
+@mixin progress-bar-color($color) {
+    &::-webkit-progress-value {
+        background-color: $color;
+    }
+
+    &::-moz-progress-bar {
+        background-color: $color;
+    }
+
+    &::-ms-fill {
+        background-color: $color;
+    }
+}
+
+:host {
+    display: block;
```

Comment:
Might want to default to inline-block, I can imagine use cases for having it inline.